src/index.js

```diff
@@ -3,7 +3,8 @@ import Sinon from 'sinon';
 import ReactWrapper from './ReactWrapper';
 import ShallowWrapper from './ShallowWrapper';
 import { onPrototype } from './Utils';
-import { renderToStaticMarkup } from './react-compat';
+import reactCompat from './react-compat';
+const {renderToStaticMarkup} = reactCompat;
```

Comment:
how easily we come to rely on linting in the editor.. I will update eslint-config-airbnb and fix linting.

Babel6 has changed the way bare objects are exported for compatability reasons meaning the properties of the exported object are actually in a `default` property.

```
TypeError: (0 , _reactCompat.renderToStaticMarkup) is not a function
      at render (index.js:8:30)
      at matchesRender (ReactWrapper-spec.js:6:9)
      at Context.<anonymous> (ReactWrapper-spec.js:540:7)
```

The alternative would be to use require:
`const { renderToStaticMarkup } = require('./react-compat').default;`

I'm not aware of any other ES6 options.